ast/marshal/json.go

```diff
@@ -0,0 +1,33 @@
+package marshal
+
+// JSONOptions defines the options for JSON operations,
+// currently only marshaling can be configured
```

Comment:
How would we configure unmarshaling? We wouldn't be able to use the struct for that, would we? 💭

Reply:
I suppose that we could if we created the type before and set the JSONOptions in it. Then `UnmarshalJSON` could read from that.

Reply:
hmm slightly uncommon but it could work 🤔